docker/package/packages.py

```diff
@@ -439,3 +439,65 @@ def mk_node_unit(
         meta=packages_meta,
     )
 )
+
+
+def mk_rollup_packages():
+    def mk_units(type, proto):
+        startup_script = "/usr/bin/tezos-rollup-node-start"
+        service_file = ServiceFile(
+            Unit(after=["network.target"], description=f"Tezos {type} rollup node"),
+            Service(
+                environment_file=f"/etc/default/tezos-{type}-rollup-node-{proto}",
+                environment=[f"PROTOCOL={proto}"],
+                exec_start_pre=[
+                    "+/usr/bin/setfacl -m u:tezos:rwx /run/systemd/ask-password"
+                ],
+                exec_start=startup_script,
+                exec_stop_post=[
+                    "+/usr/bin/setfacl -x u:tezos /run/systemd/ask-password"
+                ],
+                state_directory="tezos",
+                user="tezos",
+                type_="forking",
+                keyring_mode="shared",
+            ),
+            Install(wanted_by=["multi-user.target"]),
+        )
+        return [
+            SystemdUnit(
+                service_file=service_file,
+                startup_script=startup_script.split("/")[-1],
+                startup_script_source="tezos-rollup-node-start",
+                config_file=startup_script.split("/")[-1],
```

Comment:
```suggestion
                config_file="tezos-rollup-node.conf",
```

Reply:
i made a mistake, fixed it and forgot to stage 🙈